Lib/fontTools/pens/momentsPen.py

```diff
@@ -274,19 +274,19 @@ def _curveToOne(self, p1, p2, p3):
+		self.Planar1stMomentWrtY += r54/840 - r56/8 - r57/3 - r58*(r11 - r19)/840 + r6*(-r10*x3 + r29 + r64)/840 + r60*(r11 + 2*y3)/840 + r61*(r2 + r62 + 56*y0 + y3)/168 - r7*(10*r55 + r58 + r63)/840 + x0*(r20*x1 - r39*y0 + r43*y2 + r44 - r45*x1 + r49*x2 + r67 - r68 - r7*x3)/840 - y0*(r36*x1 + r43*x2 + 18*r52 + r58 + r65 + r66)/840
+		self.Planar2ndMomentWrtX += -r18*r72/9240 - r23*(r27 + r34 + r78)/9240 - r24*(r28*y3 + r83 + r84*x1 + r85*x3 + 210*r86 + r89)/9240 + r6*(r14*r38 + 70*r17*y2 + r20*r22 + r22*r90 + 27*r69 + 42*r71 + 54*r91*y1 + r92)/9240 - r69*r70/9240 - r7*(84*r13 - 14*r79 + r80*x3 + r82)/9240 - r71*x3/12 - r73*r74/9240 - 3*r75*(r28 + r39)/3080 - r76*(r28 + r43 + x3)/660 + r77*y3*(r11*y3 + r14 + r51)/9240 + x0*(r100*r91 + r103*y3 + r104*y2 + r105*y0 + r106*y1 + r107*r85 + r109*y1 + 135*r14*y1 + r17*r84 + r47*y3 + r50*y3 + r51*y0 + 45*r69 + 14*r71 + 126*r75 + 770*r76 + r80*y0 + r92 + r95*y2)/9240 - y0*(r100*r12 + 162*r101 + r102 + r16 + r22*r94 + r42*y3 + r74*r84 - r81*x1 - 9*r93 + r95*x2 - r96*x1 - 18*r97 + r99*y1)/9240
+		self.ProductMomentXY += -r10*(28*r113 + r114 + r115 + r41*r94 + 42*r56)/18480 - 3*r110*(r14 - r51)/220 + r111*r52*(r35 + 2*y2)/18480 - r17*r55/8 - r23*(18*r110 + 15*r52 + 7*r55)/18480 - r24*(30*r110 + r117 + r119 + r120 + r121 + r66)/18480 - r55*r73/18480 - 3*r57*y2/44 + r60*(r112*y2 + 15*r14 + 7*r17 + r4*y1 + 18*r91)/18480 + r61*(r105 + 28*r107 + r108*y1 + r109 + 2310*r24 + r51 + r80 + r85*y3 + r96 + 1260*y0*y1)/18480 + r8*(-r112*r12 + 28*r13 - r22*r37 - 27*r22*x2 + 42*r79 + r82 + 18*r86*y3 + r87*y3)/18480 + x0*(r1*y0 + 54*r101 + r102 - r103*x3 - r104*x2 - r106*x1 + r107*r25 + r125*y3 - r129*r62 + 84*r130*y2 + 81*r14*x1 - r23*x3 - r3*y0 + r31*x3 + r33*x3 + 21*r79 + r83*y3 - 84*r86*y0 + 189*r88*y2 + 60*r91*x1 + 15*r93 + 54*r97)/9240 - y0*(r110*r124 + 15*r122 + 81*r123 + r125*x1 + r125*x3 + 54*r126 + 60*r127 + r128 + 21*r56 - r60*y3 + r63*y3 + r74*r8 + 189*r86*x1)/9240
+		self.Planar2ndMomentWrtY += r111*r131/9240 - r132*r72/9240 - r132*y3/4 + 3*r133*(r30 + r48)/3080 - r52*r70*(-r35 + y2)/9240 - r55*r77*(r11 - r48)/9240 - r6*(-84*r113 - r114 - r115 - r121*y3 + 54*r127 + r38*r52 + r55*r62 + 14*r56)/9240 + r60*(-r39*y1 + r64 + r67 + r78)/9240 + r61*(r118*y3 + r124*x2 - 14*r129 - 630*r130 + r30*x3 - r77*y0 + r85*x2 + r89 + 210*x1*y2)/9240 - r7*(27*r131 + 42*r132 + 70*r134 + r135)/9240 + x0**3*(r30 + r38 + 165*y0 + y3)/660 + x0*(42*r113 + 90*r116*y3 - r117*y0 - r119*y0 - r120*y0 - r121*y0 - 9*r122 - 27*r123 - 18*r126 - 30*r127 + r128 + r136*y2 + 162*r41*x1 + r42*x3 + r54 + 63*r59*y3 - r66*y0 - r68*x3 + r99*x1)/9240 - y0*(90*r110*x1 + 45*r131 + 14*r132 + 126*r133 + 42*r134 + r135 + r136*x2 + r25*r55 + 135*r52*x1 + r65*x3)/9240
 
 if __name__ == '__main__':
 	from fontTools.misc.symfont import x, y, printGreenPen
 	printGreenPen('MomentsPen', [
 		      ('area', 1),
```

Comment:
Maybe capitalize `area` now as well?

Reply:
Well, that's breaking a test or two.

Reply:
Looks like breaking people's code loudly on `area`->`Area` is going to be loud indeed.